doc/sphinx/language/core/modules.rst

```diff
@@ -59,12 +59,12 @@ The module system provides a way of packaging related elements
 together, as well as a means of massive abstraction.
 
 
-.. cmd:: Module {? {| Import | Export } } @ident {* @module_binder } {? @of_module_type } {? := {+<+ @module_expr_inl } }
+.. cmd:: Module {? {| Import {? @import_categories } | Export {? @import_categories } } } @ident {* @module_binder } {? @of_module_type } {? := {+<+ @module_expr_inl } }
```

Comment:
@coq/doc-maintainers can I get some help to make this and others occurences say 
```suggestion
.. cmd:: Module {? {| Import | Export } {? @import_categories } } @ident {* @module_binder } {? @of_module_type } {? := {+<+ @module_expr_inl } }
```
instead?

Reply:
I pushed a commit to do that.  If you saw the following message, it's because doc_grammar isn't clever enough to match and update `{? From @dirpath } ...`.  That has to be done by hand.  You can copy the updated version from the list in `prodnCommands`(generated by doc_grammar but we aren't checking in that file.

```
Error: doc/sphinx/proof-engine/vernacular-commands.rst line 523:
NO MATCH for `{? From @dirpath } Require {? {| Import {? @import_categories } | Export {? @import_categories } } } {+ @filtered_import }`
```